cmd/contractcmd/init_validator_manager.go

```diff
@@ -63,13 +56,10 @@ func newInitValidatorManagerCmd() *cobra.Command {
 		RunE:  initValidatorManager,
 		Args:  cobrautils.ExactArgs(1),
 	}
-	networkoptions.AddNetworkFlagsToCmd(cmd, &validatorManagerFlags.Network, true, networkoptions.DefaultSupportedNetworkOptions)
-	validatorManagerFlags.PrivateKeyFlags.AddToCmd(cmd, "as contract deployer")
-	cmd.Flags().StringVar(&validatorManagerFlags.rpcEndpoint, "rpc", "", "deploy the contract into the given rpc endpoint")
-	cmd.Flags().StringSliceVar(&validatorManagerFlags.aggregatorExtraEndpoints, "aggregator-extra-endpoints", nil, "endpoints for extra nodes that are needed in signature aggregation")
-	cmd.Flags().BoolVar(&validatorManagerFlags.aggregatorAllowPrivatePeers, "aggregator-allow-private-peers", true, "allow the signature aggregator to connect to peers with private IP")
-	cmd.Flags().StringVar(&validatorManagerFlags.aggregatorLogLevel, "aggregator-log-level", constants.DefaultAggregatorLogLevel, "log level to use with signature aggregator")
-	cmd.Flags().BoolVar(&validatorManagerFlags.aggregatorLogToStdout, "aggregator-log-to-stdout", false, "dump signature aggregator logs to stdout")
+	networkoptions.AddNetworkFlagsToCmd(cmd, &network, true, networkoptions.DefaultSupportedNetworkOptions)
+	privateKeyFlags.AddToCmd(cmd, "as contract deployer")
```

Comment:
What is the "as contract deployer"?

Reply:
It is to customize the flag description 
```


func (pkf *PrivateKeyFlags) AddToCmd(
	cmd *cobra.Command,
	goal string,
) {
	pkf.fillDefaultFlagNames()
	cmd.Flags().StringVar(
		&pkf.PrivateKey,
		pkf.privateKeyFlagName,
		"",
		fmt.Sprintf("private key to use %s", goal),
	)
	cmd.Flags().StringVar(
		&pkf.KeyName,
		pkf.keyFlagName,
		"",
		fmt.Sprintf("CLI stored key to use %s", goal),
	)

```

Reply:
Would it be worth moving this into a constant or no?

Reply:
since its a one time occurrence, don't think we need to make it a constant.

Reply:
probably we can also move networkoptions and privatekey flags into flags pkg